src/main/resources/static/js/subscription.js

```diff
@@ -452,24 +452,26 @@ jQuery(document).ready(function() {
                     reload_table();
                 },
                 error : function (XMLHttpRequest, textStatus, errorThrown) {
-                    $.jGrowl("Error: " + XMLHttpRequest.responseText, {
-                        sticky : true,
-                        theme : 'Error'
-                    });
+                    reload_table();
+                    var responseJSON = JSON.parse(XMLHttpRequest.responseText);
+                    for (var i = 0; i < responseJSON.length; i++) {
+                        $.jGrowl(responseJSON[i].subscription + " :: " + responseJSON[i].reason, {sticky: true, theme: 'Error'});
+                    }
                 },
                 complete : function () {
                 }
             };
 
              $.confirm({
                  title: 'Confirm!',
-                 content: 'Are you sure you want to delete these subscriptions?<pre>' + subScriptionsToDeleteString,
+                 content: 'Are you sure you want to delete these subscriptions?<pre>' + subscriptionsToDeleteString,
                  buttons: {
                      confirm: function () {
                     	 var ajaxHttpSender = new AjaxHttpSender();
-                    	 for (i=0; i < subScriptionsToDelete.length; i++){
-                    		 ajaxHttpSender.sendAjax(frontendServiceUrl + "/subscriptions/"+subScriptionsToDelete[i], "DELETE", null, callback);
-                    	 }
+                    	 // replace all /n with comma
+                    	 subscriptionsToDeleteString = subscriptionsToDeleteString.replace(new RegExp('\n', 'g'), ',').slice(0, -1);
+                    	 console.log(subscriptionsToDeleteString);
```

Comment:
Is this log for debugging? or shall i be kept? If it is going to be kept a message before the list would be good to have:
"Sucsctions to be deleted: " + subscriptionsToDeleteString.

Reply:
Sorry. Forgot to remove.